contracts/plugins/assets/curve/CurveStableCollateral.sol

```diff
@@ -20,6 +20,9 @@ import "../curve/PoolTokens.sol";
  * ref = stablePlainPool pool invariant
  * tar = USD
  * UoA = USD
+ *
+ * @notice Curve pools with native ETH or ERC777 should be avoided,
```

Comment:
This is a nit, but technically WETH pools are a problem too if there is a bool that allows redemption in ETH

Reply:
and if we add it here it should go everywhere else we documented too, including `collateral.md`

Reply:
I've already added it to `collateral.md`, is there anywhere else I should include this?

Re: WETH, yes, I thought my statement about native ETH included that, but I'll make it more clear.